src/Fixer/Phpdoc/PhpdocToCommentFixer.php

```diff
@@ -102,6 +119,8 @@ public function configure(array $configuration = null): void
             static fn (string $tag): string => strtolower($tag),
             $this->configuration['ignored_tags']
         );
+
+        $this->allowBeforeReturnStatement = (bool) $this->configuration['allow_before_return_statement'];
```

Comment:
```suggestion
        $this->allowBeforeReturnStatement = true === $this->configuration['allow_before_return_statement'];
```
We have no casting for configuration in other fixers, let's use comparing to `true`, or even `true === $this->configuration['allow_before_return_statement']` directly in line 153.

Reply:
Fixed